src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -23,18 +23,22 @@
 using System.ComponentModel.Composition;
 using System.ComponentModel.Composition.Hosting;
 using System.ComponentModel.Composition.Primitives;
+using System.Diagnostics;
 using System.Linq;
+using FluentAssertions;
 using Moq;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests
 {
     public static class MefTestHelpers
     {
+        [DebuggerStepThrough]
```

Comment:
Miscellaneous: simplifies debugging through the tests. MS docs are [here](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.debuggerstepthroughattribute?view=net-6.0).